PyInstaller/hooks/hook-uvicorn.py

```diff
@@ -0,0 +1,43 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+"""Uvicorn hook
+
+Due to dynamic imports (see: uvicorn.config) we need this hook.
+"""
```

Comment:
Please add a link to uvicorn and which version this was tested with.

Reply:
Done as requested.